nextcord/abc.py

```diff
@@ -684,6 +685,13 @@ def permissions_for(self, obj: Union[Member, Role], /) -> Permissions:
             denied = Permissions.all_channel()
             base.value &= ~denied.value
 
+        # if you are timed out then you lose all permissions except view_channel and read_message_history
+        if obj.communication_disabled_until != None:
```

Comment:
```suggestion
        if obj.communication_disabled_until:
```

Reply:
or `is not None` for explicitness (`!=` and `==` shouldn't be used for `None` since it is a sentinel)

Reply:
I'll change it to `is not None` since I think being explicit seems like the right choice here